src/abstract_gp/abstract_gp.jl

```diff
@@ -69,8 +69,7 @@ for (m, f) in [
             "(f::AbstractGP)` is not defined (on purpose!).\n",
             "Please provide an `AbstractVector` of locations `x` at which you wish to compute your ",
             $f,
-            " vector",
-            ($m == StatsBase ? "s" : ""),
+            $((f === :mean_and_cov || f === :mean_and_var) ? " vectors" : "vector"),
```

Comment:
Ah sorry, it seems I forgot a whitespace:
```suggestion
            $((f === :mean_and_cov || f === :mean_and_var) ? " vectors" : " vector"),
```